lib/loggers/WorkerTransport.js

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright (c) 2012-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const cycle = require('cycle');
+const Transport = require('winston-transport');
+
+module.exports = class WorkerTransport extends Transport {
```

Comment:
is there anyway (maybe not now) that we could get this commented with links to the docs for `Windston Transport`. this is so future maintainers can see what API this was creates with vs whatever future version of Winston we implement down the road or other neo-logging solution as Cybernetic screaming unicorns equipped with neural link etc?